lib/dmv_facility_factory.rb

```diff
@@ -0,0 +1,49 @@
+class DmvFacilityFactory
+
+    def create_facilities(bulk_facilities)
+        state = bulk_facilities.first[:state]
+        if state == "CO"
+            build_co_facilities(bulk_facilities)
+        elsif state == "MO"
+            build_mo_facilities(bulk_facilities)
+        elsif state == "NY"
+            build_ny_facilities(bulk_facilities)
+        else
+            []
+        end
```

Comment:
I like how you put the logic into different helper methods in this control flow.